helm/logging-operator/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: logging-operator
+description: The logging-operator manages loki and promtail related config.
+home: https://github.com/giantswarm/logging-operator
+version: [[ .Version ]]
```

Comment:
Can you use abs instead of this?

Reply:
Then I'll have to put hard coded `version` and `appVersion` right ?

Reply:
yes